src/components/CardForm/CardForm.js

```diff
@@ -0,0 +1,22 @@
+import styles from './CardForm.module.scss';
+import { useState } from 'react';
+import Button from './../Button/Button';
+import TextInput from './../TextInput/TextInput';
+
+const CardForm = props => {
+    const [title, setTitle] = useState('');
+    const handleSubmit = e => {
+        e.preventDefault();
+        props.action({ title: title });
```

Comment:
Jeśli spojrzysz na definicję funkcji `addCard` w komponencie `List`, to zauważysz, że przyjmuje ona dwa parametry `newCard`, który jest obiektem i `columnId`. Natomiast tutaj przekazujesz tylko jeden argument. Musisz więc przekazać również `columnId`, który komponent `CardForm` dostaje w propsach